libcrux-ml-dsa/build.rs

```diff
@@ -0,0 +1,36 @@
+use std::env;
+
+fn main() {
+    let target_arch = env::var("CARGO_CFG_TARGET_ARCH").unwrap();
+    let disable_simd128 = read_env("LIBCRUX_DISABLE_SIMD128");
+    let disable_simd256 = read_env("LIBCRUX_DISABLE_SIMD256");
+
+    // Force a simd build. Make sure you know what you're doing.
+    let enable_simd128 = read_env("LIBCRUX_ENABLE_SIMD128");
+    let enable_simd256 = read_env("LIBCRUX_ENABLE_SIMD256");
+
+    let simd128_possible = target_arch == "aarch64";
+    if (simd128_possible || enable_simd128) && !disable_simd128 {
+        // We enable simd128 on all aarch64 builds.
+        println!("cargo:rustc-cfg=feature=\"simd128\"");
```

Comment:
I thought we would need `"cargo::rustc-cfg=feature=\"simd128\""` with double `:`, but I guess it also works like this?

Reply:
`::` is the new version. But the old things keep working. We still had some older compilers and issue with `:`. I guess we could move to `::` soonish. But we should make sure to set the MSRV (1.77) accordingly when we do.